pupil_src/shared_modules/video_capture/utils.py

```diff
@@ -218,7 +218,15 @@ def videos(self) -> Sequence[Video]:
 
     @property
     def containers(self) -> Sequence[Video]:
-        return [av.open(video.path) for video in self.videos]
+        lst = []
+        for video in self.videos:
+            try:
+                cont = av.open(video.path)
```

Comment:
Did you mean to use `check_container` here? We could change `check_container` to handle the failure case and return `None` in these cases.

```python
# Video Class
def check_container(self):
    try:
        cont = av.open(self.path)
        cont.streams.video[0]
        frame = next(cont.decode(video=0))
    except av.AVError:
        return None
    return cont

# VideoSet Class
@property
def containers(self):
    return [video.check_container() for video in self.videos]
```